src/components/ArticleComments.jsx

```diff
@@ -0,0 +1,149 @@
+import { useContext, useEffect, useState } from "react";
+import { userContext } from "../Context/UserProvider";
+import CommentCard from "./CommentCard";
+import axios from "axios";
+import ErrorPopup from "./ErrorPopup";
+import NotificationPopup from "./NotificationWindow";
+
+export default function ArticleComments({ article_id }) {
+    const [commentsStatus, setCommentsStatus] = useState(true);
+    const { user, setUser } = useContext(userContext);
+    const [postCommentErrStatus, setPostCommentErrStatus] = useState(null);
+    const [postLengthErrStatus, setPostLengthErrStatus] = useState(null);
+    const [comment, setComment] = useState("");
+    const [comments, setComments] = useState([]);
+    const [notificationStatus, setNotificationStatus] = useState(null);
+
+    useEffect(() => {
+        axios
+            .get(`https://nc-news-u90o.onrender.com/api/${article_id}/comments`)
+            .then((response) => {
+                return response.data;
+            })
+            .then((data) => {
+                setComments(data.commentsOnArticles);
+                setCommentsStatus(false);
+            });
+    }, []);
+
+    function handleChange(event) {
+        const value = event.target.value;
+        setComment(value);
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault();
+        console.log(comment.length);
+        if (comment.length < 3) {
+            setPostLengthErrStatus(true);
+        } else {
+            const commentData = {
+                body: comment,
+                article_id: article_id,
+                user: user.username,
+            };
+            axios
```

Comment:
Ideally your API calls would be separated into functions in a separate api.js file
Not mandatory but would help to organise the code a bit